workflow/cases/fcst_only.yaml

```diff
@@ -0,0 +1,8 @@
+case:
+  settings:
+    SDATE: 2011-10-01t00:00:00
+    EDATE: 2011-10-03t00:00:00
```

Comment:
Do we need 3 cycles? Would 2 be enough for testing?

Reply:
That's a good question... For the first first round I thought 3, but 2 in the future is probably sufficient.